jwst/jump/jump_step.py

```diff
@@ -16,7 +16,7 @@ class JumpStep(Step):
 
     spec = """
         rejection_threshold = float(default=4.0,min=0) # CR sigma rejection threshold
-        maximum_cores = option('quarter', 'half', 'all', default=None) # max number of processes to create
+        maximum_cores = option('one', 'quarter', 'half', 'all', default='one') # max number of processes to create
```

Comment:
I notice that in the recently merged multi-processing version of the ramp_fit step this param is defined in the spec as:
```
maximum_cores = string(default='none')
```
Down within the ramp_fit step code it does allow for additional values like "quarter", "half", and "all", as is the case here in the jump step. So I'm guessing one or the other of the two steps should be updated to make the definition of `maximum_cores` consistent between the two. Because several options are actually allowed in both steps, I'm guessing the definition in the ramp_fit step should be the one to get updated to match the option list used here.